app/src/views/home/maximize_screen.tsx

```diff
@@ -68,84 +76,122 @@ export default function MaximizeScreen({
+                  year: "numeric",
+                  month: "short",
+                  day: "numeric",
+                })}
+              </ThemedText>
+            </ThemedView>
+          </ThemedView>
+          <ThemedView
+            style={{ position: "absolute", right: 15, top: 30 }}
+            colorType="transparent"
+          >
+            {challenge.location && (
+              <ThemedIconButton
+                name="location-outline"
+                testID="location-button"
+                onPress={() => {
+                  navigation.navigate("MapScreen", {
+                    navigation: navigation,
+                    user: user,
+                    firestoreCtrl: firestoreCtrl,
+                    location: challenge.location,
+                  });
+                }}
+                size={30}
+                style={styles.locationButton}
+                colorType="white"
+              />
+            )}
           </ThemedView>
         </ThemedView>
 
-        {challenge.location && (
-          <ThemedIconButton
-            name="location-outline"
-            onPress={() => {
-              navigation.navigate("MapScreen", {
-                navigation: navigation,
-                user: user,
-                firestoreCtrl: firestoreCtrl,
-                location: challenge.location,
-              });
-            }}
-            size={25}
-            colorType="white"
-          />
-        )}
+        {/* Post Image with Double Tap */}
+        <TouchableWithoutFeedback onPress={handleDoubleTap}>
+          <ThemedView style={styles.imageContainer}>
+            <Image
+              testID="post-image"
+              source={postImage ? { uri: postImage } : require(noImage)}
+              style={styles.postImage}
+            />
+          </ThemedView>
+        </TouchableWithoutFeedback>
 
-        <ThemedView style={styles.container} colorType="transparent">
-          <Image
-            source={postImage ? { uri: postImage } : noImageUri}
-            style={styles.image}
-          />
-        </ThemedView>
+        {/* Post Description */}
+        {Boolean(postDescription) && (
+          <ThemedText style={styles.postDescription} colorType="white">
+            {postDescription}
+          </ThemedText>
+        )}
 
-        <ThemedView style={styles.likeSection} colorType="transparent">
+        {/* Like Section */}
+        <ThemedView style={styles.likeSection}>
           <ThemedIconButton
             name={isLiked ? "heart" : "heart-outline"}
             onPress={toggleLike}
-            size={35}
+            size={30}
             color={isLiked ? "red" : "white"}
             testID="like-button"
           />
-          <ThemedText style={styles.likeCountText} colorType="white">
-            {likeList.length} {likeList.length === 1 ? "like" : "likes"}
+          <ThemedText style={styles.likeCount}>
+            {likeList.length} {likeList.length === 1 ? "Like" : "Likes"}
```

Comment:
Should be
```suggestion
            {likeList.length} {likeList.length <= 1 ? "Like" : "Likes"}
```
Otherwise, no like is displayed as `0 Likes`